bakta/features/cds.py

```diff
@@ -473,4 +479,412 @@ def revise_special_cases_annotated(genome: dict, cdss: Sequence[dict]):
+
+    cds_by_hexdigest: Dict[str, dict] = orf.get_orf_dictionary(candidates)
+    uniref90_by_hexdigest: Dict[str, str] = {aa_identifier: cds['psc']['uniref90_id'] for aa_identifier, cds in
+                                             orf.get_orf_dictionary(cdss).items()
+                                             if 'psc' in cds and 'uniref90_id' in cds['psc']}
+    with blast_out_path.open() as fh:
+        root = ET.parse(fh).getroot()
+        for query in root.findall('./BlastOutput_iterations/Iteration'):
+            aa_identifier: str = query.find('Iteration_query-def').text
+            cds: dict = cds_by_hexdigest[aa_identifier]
+            extended_positions: Dict[str, Union[int, bool]] = candidates_extended_positions[aa_identifier]
+            for hit in query.findall('./Iteration_hits/Hit'):
+                cluster_id: str = hit.find('Hit_id').text
+                if cluster_id == cds['pseudo-candidate'][DB_PSC_COL_UNIREF90]:
+                    alignment: str = hit.find('Hit_hsps/Hsp/Hsp_qseq').text
+                    ref_alignment: str = hit.find('Hit_hsps/Hsp/Hsp_hseq').text
+                    alignment_start: int = int(hit.find('Hit_hsps/Hsp/Hsp_query-from').text)
+                    alignment_stop: int = int(hit.find('Hit_hsps/Hsp/Hsp_query-to').text)
+                    alignment_length: int = int(hit.find('Hit_hsps/Hsp/Hsp_align-len').text)
+
+                    if alignment_length == len(cds['aa']):  # skip non-extended genes (full match)
+                        log.debug(
+                            'No pseudogene (full match): contig=%s, start=%i, stop=%i, strand=%s',
+                            cds['contig'], cds['start'], cds['stop'], cds['strand']
+                        )
+                        continue
+
+                    cause: Dict[Union[str, int], Union[Set[int], bool]] = pseudogenization_cause(alignment,
+                                                                                                 ref_alignment,
+                                                                                                 alignment_start,
+                                                                                                 alignment_stop,
+                                                                                                 extended_positions,
+                                                                                                 cds)
+
+                    if cause[bc.FEATURE_END_5_PRIME] or cause[bc.FEATURE_END_3_PRIME]:
+                        cds[bc.PSEUDOGENE] = {'cause': convert_dict_set_values_to_list(cause),
+                                              'pseudo-candidate': cds['pseudo-candidate']}
+
+                        if is_unprocessed(uniref90_by_hexdigest, aa_identifier, cds['pseudo-candidate'][DB_PSC_COL_UNIREF90]):
+                            cds[bc.PSEUDOGENE]['qualifier'] = bc.PSEUDOGENE_UNPROCESSED
```

Comment:
Since `qualifier` is an INSDC specific term, could you please refactor this internally to 'type' whereever this occurs?